composables/useCountdown.ts

```diff
@@ -26,7 +26,6 @@ export default function useCountdown() {
 	const displayRemainingMinutes = () => {
 		const minutesIndADay = 60 * 24
 		const currentMinute = format(new Date(), 'mm')
-		console.log(currentMinute)
```

Comment:
Subsequent extracted functions reused by this could be 
`getRemainingMinutes` 
`getRemainingHours` 
...etc.
All could be tested in terms of "given a specific date, find the remaining time... " for reliable and consistent output with different scenarios


etc...

Reply:
ya good point - i think thats what paul was saying too